src/hdusd/export/mesh.py

```diff
@@ -205,40 +205,8 @@ def sync(obj_prim, obj: bpy.types.Object, mesh: bpy.types.Mesh = None, **kwargs)
         return
 
     stage = obj_prim.GetStage()
-    parent_prim = None
-    parent_object = None
-
-    if obj.parent is not None and obj_prim.GetName() != sdf_name(obj.original):
-        parent_object = obj.original
-        parent_prim = stage.GetPrimAtPath(f"/{sdf_name(obj.original)}")
-
-    if parent_prim is not None and not parent_prim.IsValid():
-        xform = UsdGeom.Xform.Define(stage, f"/{sdf_name(obj.original)}")
-        parent_prim = xform.GetPrim()
-        xform.MakeMatrixXform().Set(Gf.Matrix4d(parent_object.matrix_world.transposed()))
-        sync(parent_prim, parent_object)
-
-    if parent_prim is not None and parent_prim.IsValid() and parent_prim.GetChildren():
-        for child in parent_prim.GetChildren():
-            if child.GetTypeName() == 'Mesh':
-                usd_mesh = UsdGeom.Mesh.Define(stage, obj_prim.GetPath().AppendChild(sdf_name(obj)))
-                usd_mesh.GetPrim().GetReferences().AddInternalReference(child.GetPath())
-
-            if child.GetTypeName() == 'Material':
-                usd_mesh = UsdGeom.Mesh.Get(stage, obj_prim.GetPath().AppendChild(sdf_name(obj)))
-                usd_material = UsdShade.Material.Get(stage, child.GetPath())
-                UsdShade.MaterialBindingAPI(usd_mesh).Bind(usd_material)
-                
-        return
-
-    original_prim = stage.GetPrimAtPath(f"/{sdf_name(obj.original)}")
-    if original_prim and original_prim.IsValid():
-        for child in original_prim.GetChildren():
-            if len(child.GetAuthoredPropertyNames()) > 0:
-                return
```

Comment:
Did you completely test this part, that your are going to remove it? Won't it break something else?

Reply:
Yes